react/features/room-lock/middleware.js

```diff
@@ -27,8 +29,7 @@ MiddlewareRegistry.register(store => next => action => {
     case CONFERENCE_FAILED: {
         const { conference, error } = action;
 
-        if (conference
```

Comment:
I'll try to clarify:

1. Imagine you have the dialog open and jicofo goes down, that's an unrecoverable error which happens while the dialog is open
2. That's a good point. It makes sense indeed. In the above scenario, there will be a time when that error **is** recoverable. I'll change it to dispatch always, in case of any other error.
3. Conference will always be there, AFAICT. Many other places in the code don't check for this, so I thought we were being overzealous here. Now that I think of it, however, there are (or will be) cases in which CONFERENCE_FAILED will have no conference: when we artificially generate it for the SDK, so I'll add back the check.